### ISSUE TITLE
exporting to html & inline graphics does not respect font family

### ISSUE THREAD

**Issue by valentas-kurauskas** [IRkernel/IRkernel#379]
IRKernel (Ubuntu 14.04, Firefox 47.0, Jupyter 4.2.1), when plotting inline does not seem to respect font family and other parameters:

    library(ggplot2)
    ggplot()+geom_point(data=data.frame(first_variable=1:10, second_variable=10:1), 
                        aes(x=first_variable,y=second_variable)) + 
    theme(text=element_text(size=19, family="mono"))

![font_family_bug](https://cloud.githubusercontent.com/assets/11426795/16809135/cdecd206-4927-11e6-89a2-8e4c9c03e3d3.png)

Even worse, when I try to export to html - the labels are scrambled whether or not I change theme().. 
The plot looks correctly when I run this in RStudio or if I save this to png/svg in the notebook.

![font_family_bug2](https://cloud.githubusercontent.com/assets/11426795/16809252/4a5b96e2-4928-11e6-802d-c8c3cee027d4.png)

**Comment by janschulz** [comment]
Which verison of irkernel, irdisplay and repr? We recently changed that only png plots are made, maybe the isolated svg (=own iframe just for the svg tag) was the problem here?

**Comment by valentas-kurauskas** [comment]
Rdisplay_0.3 repr_0.7.99   IRkernel_0.5

**Comment by janschulz** [comment]
Just for the record: I don't have that problem when I run the above code (win7, Chrome):

```R
> sessionInfo(package= c("IRkernel", "IRdisplay", "repr"))
[...]
other attached packages:
[1] IRkernel_0.6    IRdisplay_0.4.1 repr_0.7.99   
```
(Everything from master)

**Comment by takluyver** [comment]
The issue you're seeing with nbconvert certainly looks like the isolated svgs issue (jupyter/nbconvert#129), so I'd guess that you were seeing an SVG in the live notebook too.

**Comment by valentas-kurauskas** [comment]
Yes, thanks, I was seeing SVG. The solution from https://github.com/IRkernel/IRkernel/issues/331 solves the export issue. I'm still suck with the default font family though.
